src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +51,45 @@
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
```

Comment:
Ensure that the `injector.getInstance(MovieSessionService.class)` returns an instance of `MovieSessionService` to avoid `ClassCastException`.